src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
```

Comment:
This method appears to not exist on `SVGElement`, but [rather on `SVGTextContentElement`](https://developer.mozilla.org/en-US/docs/Web/API/SVGTextContentElement). Can we please move these to a (new) `SVGTextContentElement` class that extends (a new) `MockSVGGraphicsElement` that extends `MockSVGElement`? The reason being is that these methods shouldn't be available on MockSVGElement (and by extension, any of its child classes except `SVGTextContentElement`)